docs/schemas/2024-04/donor-details.json

```
@@ -212,7 +212,16 @@
         }
       ],
       "type": "object",
-      "required": ["dateOfBirth", "email", "status"],
+      "required": ["dateOfBirth", "email", "status", "channel"],
```

Presumably want to remove `email` from the default required set?
```suggestion
      "required": ["dateOfBirth", "status", "channel"],
```

---

Missed this - yep agree, I'll remove